projects/baby_driver/scripts/message_defs.py

```diff
@@ -0,0 +1,19 @@
+"""Constants for the babydriver CAN protocol."""
```

Comment:
do we want to generate this file via codegen? protobufs are pretty cool (could also leave this as an improvement to make later (the more tickets the merrier right))